lib/getDeterministricAttributes.tsx

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable */
```

Comment:
why are we disabling ES-lint for the entire file?
- if you need to disable es-lint, please only disable a `specific line` and a `specific error`